EIPS/eip-1193.md

```diff
@@ -366,7 +376,13 @@ All supported RPC methods **MUST** be identified by unique strings.
 
 Providers **MAY** support whatever RPC methods required to fulfill their purpose, standardized or otherwise.
 
-If a Provider supports a method defined in a finalized EIP, the Provider's implementation of this method **MUST** adhere to the method's specification.
+If a Provider supports a method defined in a finalized EIP, the Provider's implementation of this method **MUST** adhere to the method's specification. This includes:
+
+- The method name, given as the string value of `RequestArguments.method`
+- The method parameters, given as the value of `RequestArguments.params`, either:
+  - By position, in an array
+  - By name, in an object
```

Comment:
This feels like it is redundant with the initial specification.  Earlier in the specification we have already asserted that `method`'s type is `string` and `params` type is `unknown[] | object | undefined`.

Reply:
Very good. I removed all references to types from these lines, and just explicitly state how the RPC method specification relates to the fields of the `RequestArguments` interface.

Unless I'm missing something, that was previously only implied, and I'd rather it be explicit.

Reply:
I removed this from the `Supported RPC Methods` section and added a more concise version to the `request` section, which has been made more concise overall.